src/components/Transaction/TransactionCard/index.tsx

```diff
@@ -65,7 +65,7 @@ const CardCell = ({ input, address }: { input: InputOutput; address?: string })
   let highLight = false
   if (input.address_hash) {
     addressText = startEndEllipsis(input.address_hash, 12)
-    if (address && input.address_hash !== address) {
+    if (input.address_hash !== address) {
```

Comment:
Why delete `address` condition ?

Reply:
Clicking on the address in the BlockDetail page also takes you to the Address page.

Reply:
`address === input.address_hash`, address should not be clickable

Reply:
I see. But I think we could not remove address condition. I suggest:

```
if (!address) hightlight = true
else if (input.address_hash === address) highlight = true
```

not just delete address condition.